### PR TITLE
V1300944: Converting commands to codebox to avoid localization issues

### PR DESCRIPTION
**Pull request by v-maudel**
Hello, @bricelam,
Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  
"Unescaped commands are generating conflicts with translated pages:
https://docs.microsoft.com/fr-fr/ef/core/miscellaneous/cli/dotnet#using-the-tools
https://docs.microsoft.com/ja-jp/ef/core/miscellaneous/cli/dotnet#using-the-tools
"
Please review and merge the proposed file change to fix to target versions. If you make related fix in another PR  then share your PR number so we can confirm and close this PR.
Many thanks in advance.

### CODE REVIEW

File: entity-framework/core/miscellaneous/cli/dotnet.md [hunk 1]
```diff
@@ -64,10 +64,10 @@ Using the tools
 Whenever you invoke a command, there are two projects involved:
 
 The target project is where any files are added (or in some cases removed). The target project defaults to the project
-in the current directory, but can be changed using the <nobr>**--project**</nobr> option.
+in the current directory, but can be changed using the <**`--project`**</nobr> option.
```
Comment (divega) [inline entity-framework/core/miscellaneous/cli/dotnet.md]: @v-maudel isn't the '<' and `</nobr>` supposed to be removed here?

Comment (divega) [reply entity-framework/core/miscellaneous/cli/dotnet.md]: Or maybe `nobrb>` which was removed should be kept?

**Comment by v-maudel** [comment]
Hi @divega , I shouldn't delete the nobr tag, I've reverted it. Please, let me know if this is fine now. 

regards

**Comment by divega** [comment]
Thanks @v-maudel. I will let @bricelam review the rest of the changes.

**Comment by bricelam** [comment]
I don't like that the formatting has to change to facilitate localization, but if it's the only way ... :shipit:

**Comment by v-maudel** [comment]
Thanks @divega and @bricelam . 
Actually I don't like this formatting either but is the only solution we have so far. 

regards.

**Pull request merged**